server/runtime/src/main/java/org/infinispan/server/tasks/DistributedServerTaskRunner.java

```diff
@@ -36,8 +38,9 @@ public DistributedServerTaskRunner(ServerTaskEngine serverTaskEngine) {
             results.add(v);
          }
       };
-      CompletableFuture<Void> future = clusterExecutor.submitConsumer(new DistributedServerTask<>(
-            masterCacheNode.getName(), taskName, context.getParameters()), triConsumer);
+      List<TaskParameter> taskParams = context.getParameters().orElse(Collections.emptyMap()).entrySet().stream().map(e -> new TaskParameter(e.getKey(), e.getValue().toString())).collect(Collectors.toList());
```

Comment:
Class variable `serverTaskEngine` is never used, so we can remove the constructor.

Reply:
Removed